src/backend/distributed/commands/table.c

```diff
@@ -50,6 +55,12 @@ static void ErrorIfAlterTableDefinesFKeyFromPostgresToCitusLocalTable(
 static List * GetAlterTableStmtFKeyConstraintList(AlterTableStmt *alterTableStatement);
 static List * GetAlterTableCommandFKeyConstraintList(AlterTableCmd *command);
 static bool AlterTableCommandTypeIsTrigger(AlterTableType alterTableType);
+static bool AlterTableHasCommandByFunc(AlterTableStmt *alterTableStatement,
```

Comment:
maybe `AlterTableCmdCheckFunction `  or so to be inlined with `FindNodeMatchingCheckFunction`

Reply:
makes sense, will do this change